cfgov/cfgov/settings/production.py

```diff
@@ -185,8 +185,30 @@
 SESSION_EXPIRE_AT_BROWSER_CLOSE = True
 CSRF_COOKIE_HTTPONLY = True
 CSRF_COOKIE_SECURE = True
-SECURE_BROWSER_XSS_FILTER = True
 SECURE_HSTS_SECONDS = 600
 SECURE_HSTS_INCLUDE_SUBDOMAINS = True
 SECURE_CONTENT_TYPE_NOSNIFF = True
 SECURE_HSTS_PRELOAD = True
+SECURE_SSL_REDIRECT = True
+
+# Require the SECRET_KEY as an environment variable
+try:
+    SECRET_KEY = os.environ["SECRET_KEY"]
+except KeyError:
+    raise ImproperlyConfigured(
+        "The SECRET_KEY environment variable must be set"
+    )
```

Comment:
Currently this passes if `SECRET_KEY` is set but empty, e.g. `SECRET_KEY= cfgov/manage.py ...`.

```suggestion
SECRET_KEY = os.getenv("SECRET_KEY")

if not SECRET_KEY:
    raise ImproperlyConfigured(
        "The SECRET_KEY environment variable must be set"
    )
```

or if you really want to be "modern":

```py
if not (SECRET_KEY := os.getenv("SECRET_KEY")):
   ...
```